test/test_field.jl

```diff
@@ -351,7 +349,19 @@ end
         @info "  Testing field interpolation..."
 
         for arch in archs, FT in float_types
-            run_field_interpolation_tests(FT, arch)
+            reg_grid = RectilinearGrid(arch, size=(4, 5, 7), x=(0, 1), y=(-π, π), z=(-5.3, 2.7), halo=(1, 1, 1))

```

Comment:
this grid does not use the `FT`; was this change on purpose?

```suggestion
            reg_grid = RectilinearGrid(arch, FT, size=(4, 5, 7), x=(0, 1), y=(-π, π), z=(-5.3, 2.7), halo=(1, 1, 1))

```